index.scss

```diff
@@ -0,0 +1,72 @@
+$main-background-color: #ECEFF8;
+
+$navbar-color: #4F9CC8;
+
+$card-background-color: #9DBED4;
+
+@mixin centering-container {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+
+* {
+  margin: 0;
+  font-family: "Trebuchet MS", sans-serif;
+  box-sizing: border-box;
+}
+
+body{
+  background-color: $main-background-color;
+  nav {
+    background-color: $navbar-color;
+    padding: 10px;
+    position: fixed;
+    width: 100vw;
+    z-index: 10;
+    }
+    h1{
+      padding: 15px 0px 15px 10px;
+      color: white;
+    }
+  .allUserCardContainer {
+    @include centering-container;
+    width: 50vw;
```

Comment:
why exactly 50vw? also i notice this is set in 2 places, wouldn't it be better if it was only in 1 place?